server/operator.go

```diff
@@ -77,24 +79,82 @@ func ParseResourceKind(name string) ResourceKind {
 	return UnKnownKind
 }
 
+// OperatorState indicates state of the operator
+type OperatorState int
+
+const (
+	// OperatorUnKnownState indicates the unknown state
+	OperatorUnKnownState OperatorState = iota
+	// OperatorDoing indicates the doing state
+	OperatorDoing
```

Comment:
Or _Wait_ as the initial state, after the operator is sent to tikv, update to _Running_.